15:43:51.381 INFOS [org.apache.catalina.core.ApplicationContext log] - Initializing Spring root WebApplicationContext  
15:44:17.816 INFOS [org.apache.catalina.core.ApplicationContext log] - Initializing Spring root WebApplicationContext  
15:44:25.491 INFOS [org.apache.catalina.core.ApplicationContext log] - ContextListener: contextInitialized()  
15:44:25.491 INFOS [org.apache.catalina.core.ApplicationContext log] - SessionListener: contextInitialized()  
15:46:20.581 INFOS [org.apache.catalina.core.ApplicationContext log] - Closing Spring root WebApplicationContext  
15:46:23.623 INFOS [org.apache.catalina.core.ApplicationContext log] - SessionListener: contextDestroyed()  
15:46:23.623 INFOS [org.apache.catalina.core.ApplicationContext log] - ContextListener: contextDestroyed()  
15:46:23.639 INFOS [org.apache.catalina.core.ApplicationContext log] - Closing Spring root WebApplicationContext  
15:46:34.387 INFOS [org.apache.catalina.core.ApplicationContext log] - Initializing Spring root WebApplicationContext  
15:46:38.240 INFOS [org.apache.catalina.core.ApplicationContext log] - Initializing Spring root WebApplicationContext  
15:46:40.149 INFOS [org.apache.catalina.core.ApplicationContext log] - ContextListener: contextInitialized()  
15:46:40.149 INFOS [org.apache.catalina.core.ApplicationContext log] - SessionListener: contextInitialized()  
15:47:48.967 GRAVE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
15:51:03.762 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
15:51:03.770 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
18:59:40.866 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
18:59:41.100 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
18:59:41.180 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
18:59:41.416 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
18:59:54.693 SEVERE [org.apache.catalina.core.StandardWrapperValve invoke] - "Servlet.service()" pour la servlet ActionServlet a généré une exception 
java.lang.IllegalStateException: "getOutputStream()" a déjà été appelé pour cette réponse
	at org.apache.catalina.connector.Response.getWriter(Response.java:611)
	at org.apache.catalina.connector.ResponseFacade.getWriter(ResponseFacade.java:198)
	at quartisweb.filter.FiltreSession.doFilter(FiltreSession.java:333)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:235)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:127)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:298)
	at org.apache.coyote.http11.Http11NioProcessor.process(Http11NioProcessor.java:886)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:721)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:2256)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
 
